rosdep/python.yaml

```
@@ -5858,6 +5858,8 @@ python3-rafcon-pip:
   ubuntu:
     pip:
       packages: [rafcon]
+python3-rdflib:
+  ubuntu: [python3-rdflib]
```

This package is available on many platforms. As outlined in the [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#rosdep-rules-contributions), we'd like to see due diligence to investigate the other platforms and add rules for as many as possible. Please do so in the future.

- arch: https://www.archlinux.org/packages/extra/any/python-rdflib/
- debian: https://packages.debian.org/buster/python3-rdflib
- fedora: https://apps.fedoraproject.org/packages/python3-rdflib
- gentoo: https://packages.gentoo.org/packages/dev-python/rdflib
- rhel: https://apps.fedoraproject.org/packages/python3-rdflib

```suggestion
python3-rdflib:
  arch: [python-rdflib]
  debian: [python3-rdflib]
  fedora: [python3-rdflib]
  gentoo: [dev-python/rdflib]
  rhel:
    '8': ['python%{python3_pkgversion}-rdflib']
  ubuntu: [python3-rdflib]
```